image/php-fpm/Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM ubuntu
+
+# ADD /laravel /var/www
```

Comment:
In Docker 1.5 (newer than the issue you linked), support for multiple Dockerfiles per build context was added.  You and I spoke about this in the office, so I think we should be re-adding this line and  each container should ship with its own copy of code. :)